python/kfserving/test/test_v1beta1_predictor_protocols.py

```diff
@@ -57,11 +57,13 @@ def make_instance(self, include_optional):
                     default_gpu_image_version="0",
                     default_image_version="0",
                     image="0",
+                    supported_frameworks=['0']
```

Comment:
What is the logic behind the if statement here, as I understand the result is the same(no mater my current change) @yuzisun